web/src/app/browser/src/hardwareEventKeyboard.ts

```diff
@@ -335,7 +335,7 @@ export default class HardwareEventKeyboard extends HardKeyboard {
   _KeyUp: (e: KeyboardEvent) => boolean = (e) => {
     const target = eventOutputTarget(e);
     var Levent = preprocessKeyboardEvent(e, this.processor, this.hardDevice);
-    if(Levent == null) {
+    if(Levent == null || target == null) {
```

Comment:
If `target == null` here, that in itself is an error we should probably log in some manner.  The keystroke handler should only activate for elements that are currently in an attached & enabled state.

The attachment engine is supposed to ensure that the handler is only set for such elements, removing an element's event listeners when not attached.  I feel like this change only masks the actual error.

Reply:
So should I remove the `target == null` check again so that we see the error on the console? Or how should we log?